@xen-orchestra/web/src/components/site/pools/PoolsTable.vue

```diff
@@ -0,0 +1,271 @@
+<template>
+  <div class="site-pools-table">
+    <UiTitle>
+      {{ t('pools') }}
+    </UiTitle>
+    <div class="container">
+      <div class="table-actions">
+        <UiQuerySearchBar @search="value => (searchQuery = value)" />
+        <UiTableActions :title="t('table-actions')">
+          <UiButton
+            v-tooltip="t('coming-soon')"
+            disabled
+            :left-icon="faCaretSquareDown"
+            variant="tertiary"
+            accent="brand"
+            size="medium"
+          >
+            {{ t('change-state') }}
+          </UiButton>
+          <UiButton
+            v-tooltip="t('coming-soon')"
+            disabled
+            :left-icon="faEdit"
+            variant="tertiary"
+            accent="brand"
+            size="medium"
+          >
+            {{ t('edit') }}
+          </UiButton>
+          <UiButton
+            v-tooltip="t('coming-soon')"
+            disabled
+            :left-icon="faEraser"
+            variant="tertiary"
+            accent="danger"
+            size="medium"
+          >
+            {{ t('forget') }}
+          </UiButton>
+        </UiTableActions>
+        <UiTopBottomTable :selected-items="0" :total-items="0" @toggle-select-all="toggleSelect" />
+      </div>
+      <VtsDataTable
+        :is-ready="isServerReady && isHostReady"
```

Comment:
```suggestion
        :is-ready="isServerReady"
```
I'm not sure we want to wait until the hosts are ready (because if they aren't that means we're not displaying anything at all?)